pyincore/analyses/housingrecovery/housingrecovery.py

```diff
@@ -0,0 +1,396 @@
+
+    Args:
+        incore_client (IncoreClient): Service authentication.
+
+    """
+    def __init__(self, incore_client):
+        super(HousingRecovery, self).__init__(incore_client)
+
+    def get_spec(self):
+        return {
+            "name": "housing-recovery",
+            "description": "Housing Recovery Analysis",
+            "input_parameters": [
+                {
+                    "id": "base_year",
+                    "required": False,
+                    "description": "Base year is used to calculate improvement age. It needs to be set to the tax "
+                                   "assessment year representing pre-disaster building values. For example for GCAD "
+                                   "data which represents improvement valuation before Hurricane Ike impacts."
+                                   "Deafult 2008",
+                    "type": int
+                },
+                {
+                    "id": "fips",
+                    "required": False,
+                    "description": "A county FIPS Code is a five-digit number used to designate a specific county.",
+                    "type": str
+                },
+                {
+                   "id": "result_name",
+                   "required": True,
+                   "description": "Result CSV dataset name",
+                   "type": str
+                }
+            ],
+            "input_datasets": [
+                {
+                   "id": "population_dislocation",
+                   "required": True,
+                   "description": "Population Dislocation aggregated to the block group level",
+                   "type": ["incore:popDislocation"]
+                },
+                {
+                   "id": "building_area",
+                   "required": True,
+                   "description": "Building square footage and damage. Damage is the actual building value loss "
+                                  "in percentage terms observed through the County Appraisal District (GCAD) data",
+                   "type": ["incore:buildingInventoryArea"]
+                },
+                {
+                   "id": "census_block_groups_data",
+                   "required": True,
+                   "description": "Census ACS data, 2010 5yr data for block groups available at IPUMS NHGIS "
+                                  "web site.",
+                   "type": ["incore:censusBlockGroupsData"]
+                },
+                {
+                   "id": "census_appraisal_data",
+                   "required": False,
+                   "description": "Census data, 2010 Decennial Census District (GCAD) Census data",
+                   "type": ["incore:censusAppraisalData"]
+                }
+            ],
+            "output_datasets": [
+                {
+                   "id": "result",
+                   "description": "A csv file with the building values for the 6 years following the disaster"
```

Comment:
can you update the description to say that it also returns what `hse_rec_fin_index` represents, so it's clear what the `index_n` columns mean in the outputs. 

Might also be helpful to say year -1 represents pre-impact conditions

Reply:
Done